src/manifest.jl

```diff
@@ -133,7 +133,7 @@ end
 function Manifest(raw::Dict, f_or_io::Union{String, IO})::Manifest
     julia_version = haskey(raw, "julia_version") ? VersionNumber(raw["julia_version"]) : nothing
     manifest_format = VersionNumber(raw["manifest_format"])
```

Comment:
```suggestion
    manifest_format = VersionNumber(raw["manifest_format"]::String)
```

Might also do it?

Reply:
Sounds reasonable. I changed it in cd830cb